<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Investigating dead ends in the vessel network</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonAmFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Input network file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAmFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>371</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelOutputFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonImageFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output image file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiffFile">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxVoxel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_zvoxel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_z">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_xvoxel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_x">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_yvoxel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_y">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_voxel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Voxel Size (um)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_margin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Margin thickness (um)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMargin">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonGo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAmFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>amFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonImageFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>tiffFileSelecter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_xvoxel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>dataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_yvoxel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>dataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditMargin</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>dataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonGo</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>am_tiffer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_zvoxel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>dataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>amFileSelecter()</slot>
  <slot>tiffFileSelecter()</slot>
  <slot>dataChanged()</slot>
  <slot>am_tiffer()</slot>
 </slots>
</ui>
